.rubocop.yml

```diff
@@ -1,7 +1,9 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
-  TargetRubyVersion: 2.0
+  TargetRubyVersion: 2.4
```

Comment:
I don't think smart-proxy is on 2.5+, yet, or was it already SCLed?

Reply:
Oh thanks, rebased this was an accident.